.github/workflows/rpm_build.yml

```diff
@@ -0,0 +1,37 @@
+name: RPM Build
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: centos:8
+      env:
+        NODE_ENV: development
+      ports:
+        - 80
+      options: --cpus 1
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: environment setup
+        run: |
+          dnf -y --disablerepo '*' --enablerepo=extras swap centos-linux-repos centos-stream-repos
+          dnf -y distro-sync
+          dnf -y makecache --refresh
+          dnf install -y rpm-build rpmdevtools git make
+          dnf module -y install go-toolset 
+          rpmdev-setuptree
+          tar -czf /github/home/rpmbuild/SOURCES/nnf-datamovement-1.0.tar.gz --transform 's,^,nnf-datamovement-1.0/,' .
+      - name: build rpms
+        run: rpmbuild -ba daemons/compute/server/nnf-dm.spec
+      - name: upload rpms
+        uses: actions/upload-artifact@v3
+        with:
+          name: nnf-datamovement-1.0-1.el8.x86_64.rpm
```

Comment:
do we need to be worried about these version numbers changing?

Reply:
Yeah, if we change it, we'll have to change it in a few different areas